docs/templates/language-reference.md

```diff
@@ -44,20 +44,45 @@ Ideally, also with a full working example that shows it in context.
 
 Depending on the language, a brief description of the recommended way to handle dependencies in the language.
 
-## Connect to services
+## Connecting to services
 
-If available, include examples for connecting in an app
+If available, include examples for connecting to different services in an app
 (taken from https://examples.docs.platform.sh/).
 
-### Configuration reader
+The following examples show how to use <LANGUAGE_NAME> to access various services. To configure a given service, see the page dedicated to that service.
```

Comment:
:hammer_and_wrench: 
```suggestion
The following examples show how to use <LANGUAGE_NAME> to access various [services](/add-services/_index.md).
To configure a given service, see the page dedicated to that service.
```

:lady_beetle: I would note that this addition means the text in the template is more of a mixture of instructions and text that could be included in the final version and it's not immediately clear which is which.